application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -70,6 +70,24 @@ interface PlaceInterface {
   lon: Float
 }
 
+"Interface for rental vehicles, e.g. free floating vehicles and rental vehicle stations"
+interface VehicleRental {
+  "If true, a vehicle can be picked up."
+  allowPickupNow: Boolean
+  "Latitude of the vehicle rental object (WGS 84)"
+  lat: Float
+  "Longitude of the vehicle rental object (WGS 84)"
+  lon: Float
+  "Name of the vehicle"
```

Comment:
```suggestion
  "Name of the vehicle or station."
```

Reply:
Actually, I'm not sure if the name should be part of this interface, in the context of rental vehicles, name is slightly weird.

Reply:
I think you're right. Our internal model has a name for a free-floating vehicle but that is probably a mistake.

GBFS doesn't have it in free_bike_status: https://github.com/MobilityData/gbfs/blob/v2.3/gbfs.md#free_bike_statusjson

Reply:
You're right! 
I'll remove the `name` property from the RentalVehicle interface then and I'll keep it only in the `VehicleRentalStation` type. 
I think the confusion arises from the `VehicleRentalPlace` interface where there are some get methods that belongs only to the `VehicleRentalStation` class implementation and not the `VehicleRentalVehicle` (free-floating vehicle). 
Some of those are:
- getCapacity: it's always 1 in the free-floating vehicle implementation
- getVehiclesAvailable: also 1
- allowDropoffNow: always false in a free-floating
Maybe I can refactor this interface in a separate PR

Reply:
While trying to remove it from `VehicleRentalPlace.java` I noticed that the *getName* method is being used in multiple other places (like [here](https://github.com/JustCris654/OpenTripPlanner/blob/ccc614041f838cb1f23d3c133f1fb9830b3f5e6c/application/src/main/java/org/opentripplanner/service/vehiclerental/street/VehicleRentalPlaceVertex.java#L24)), I don't think I have knowledge to remove it from all this places.
What's the best way to proceed? I could remove it from the gtfs graphl API but leave it in the `VehicleRentalPlace` interface.

Reply:
Unfortunately, we have to live with the existence of the rental vehicle name for backwards compatibility reasons. Therefore, in the context of this pr, it's enough that you just remove the name from the new GraphQL interface, you shouldn't remove it from the RentalVehicle type or from elsewhere.